sdk/core/core-amqp/package.json

```diff
@@ -35,29 +35,29 @@
   ],
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:samples": "echo skipped",
+    "build:samples": "echo Obsolete",
```

Comment:
can we just delete this task everywhere?

Reply:
I believe we need to keep it because some packages did not do the migration to samples v2 yet and this is the only way for their samples to get built in CI. @witemple-msft can you please put removing this script in your to-do list/epic once the migration finished across all packages?